cli/args/flags.rs

```diff
@@ -992,6 +994,8 @@ fn clap_root() -> Command {
   }
```

Comment:
Would be nice if the unstable granular flags were sorted alphabetically.

Reply:
I think you could do it in place here?